projects/packages/forms/src/class-jetpack-forms.php

```diff
@@ -62,4 +62,14 @@ public static function is_feedback_dashboard_enabled() {
 		 */
 		return apply_filters( 'jetpack_forms_dashboard_enable', false );
 	}
+
+	/**
+	 * Initialization, calls for register pattern and loads any extra file/libs.
+	 */
+	public static function init() {
+		Util::register_pattern();
+
+		// Load any extra file/libs so they are available on the namespace for use (Forms\Post_To_URL_Hook::initialize)
+		require_once __DIR__ . '/class-post-to-url-hook.php';
```

Comment:
I don't understand why we need this at all? The file should already be loaded through the autoloader and you can just proceed to use `Automattic\Jetpack\Forms\Post_To_Url_Hook` where necessary.

Reply:
Right! I was installing the package with no success, then I realized I needed to install the plugin for the autoloader to update. Thanks!